### ISSUE TITLE
Error message when opening xlsx generated with EPPlus in Excel (Removed part: /xl/vbaProject.bin)

### ISSUE THREAD

**Issue by dantronik-markus** [EPPlusSoftware/EPPlus#729]
After opening an Excel template, adding some data and saving it as an xslm file, I keep getting the following error message upon opening:
`Removed part: Part /xl/vbaProject.bin. (Visual Basic for Applications (VBA))`

The xlst file can be opened with Excel just fine.

The generated Excel file is as follows:
[dfb03914f2964d06854edae67c6f360f.zip](https://github.com/EPPlusSoftware/EPPlus/files/9821805/dfb03914f2964d06854edae67c6f360f.zip)

The two relevant files that contains most of our source code are here:
[EPPlusBaseSourceCode.zip](https://github.com/EPPlusSoftware/EPPlus/files/9821830/EPPlusBaseSourceCode.zip)

We're using EPPlus 6.0.8

**Comment by dantronik-markus** [comment]
The template we load with it is the following:
[OFF103225STHFBericht.zip](https://github.com/EPPlusSoftware/EPPlus/files/9821855/OFF103225STHFBericht.zip)

**Comment by JanKallman** [comment]
Opening and saving the template seems to work. I can't use the code attached. Please provide a test method to fully reproducible this issue.

**Comment by dantronik-markus** [comment]
[EPPlusIssue729Test.zip](https://github.com/EPPlusSoftware/EPPlus/files/9830677/EPPlusIssue729Test.zip)
I've extracted the code that reproduces the issue and created a sample project.

**Comment by JanKallman** [comment]
This issue should now be fixed. After a lot of investigation, I found that if a VBA project contains a Form that does not have a "Package" Guid set for the designer in the Project Stream within the VBAProject.bin in the package, EPPlus wrote an empty Package entry to the stream. This caused the workbook to become corrupt. 
I have committed a fix that should solve this issue. Feel free to try it using our develop nuget feed...
https://github.com/EPPlusSoftware/EPPlus/wiki/Using-our-Develop-Nuget-Feed

Package: 6.0.8.1067-20221026-develop
